### DISCUSSION TITLE
ONNX Runtime 1.15 Release Candidate available for testing

### DISCUSSION THREAD

**Discussion by snnn** [microsoft/onnxruntime#15813] [General]
ONNX Runtime 1.15 will be released later this month. Release candidate builds are available now for testing. If you encounter issues, please report them by responding in this issue.

Release branch: [rel-1.15.0](https://github.com/microsoft/onnxruntime/tree/rel-1.15.0)
Release manager: @snnn
|                | Package Name                                     | Version                                                                                                                    |
|----------------|--------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------|
|     Python     |     ort-nightly                                  |     1.15.0.dev20230502003                                                                                                  |
|                |     ort-nightly-gpu                              |     1.15.0.dev20230502003                                                                                                  |
|                |     ort-nightly-directml                         |     1.15.0.dev20230502003                                                                                                  |
|                |     ort-training                                 |     1.15.0.dev20230502001                                                                                                  |
|     Nuget      |     Microsoft.ML.OnnxRuntime                     |     1.15.0-dev-20230503-1203-abdd4f518a                                                                                    |
|                |     Microsoft.ML.OnnxRuntime.Training            |     1.15.0-dev-20230503-1222-abdd4f518a                                                                                    |
|                |     Microsoft.ML.OnnxRuntime.Gpu                 |     1.15.0-dev-20230503-0741-abdd4f518a                                                                                    |
|                |     Microsoft.ML.OnnxRuntime.Managed             |     1.15.0-dev-20230503-0741-abdd4f518a     1.15.0-dev-20230503-1203-abdd4f518a     1.15.0-dev-20230503-1222-abdd4f518a    |
|                |     Microsoft.ML.OnnxRuntime.DirectML            |     1.15.0-dev-20230502-1020-abdd4f518a                                                                                    |
|     Java       |     com.microsoft.onnxruntime:onnxruntime        |     1.15.0-rc1                                                                                                             |
|                |     com.microsoft.onnxruntime:onnxruntime_gpu    |      1.15.0-rc1                                                                                                                        |
|     NPM        |     onnxruntime-web                              |     1.15.0-dev.20230502-abdd4f518a                                                                                         |

All python, nuget and Jave packages are available at https://aiinfra.visualstudio.com/PublicPackages/_artifacts/feed/ORT-Nightly

For example, you may install the "ort-nightly" python package in a way like:

python -m pip install coloredlogs flatbuffers numpy packaging protobuf sympy
python -m pip install -i https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/ORT-Nightly/pypi/simple/ ort_nightly==1.15.0.dev20230502003            (Change the package name or version as needed)

The NPM test packages are available at www.npmjs.com. You can use the following links to find them:

1. https://www.npmjs.com/package/onnxruntime-web
2. https://www.npmjs.com/package/onnxruntime-common
3. https://www.npmjs.com/package/onnxruntime-node
4. https://www.npmjs.com/package/onnxruntime-react-native

**Comment by snnn** [comment]
ONNX 1.14 just released. https://github.com/onnx/onnx/discussions/5203 . We will bring the tag to our 1.15 release.

**Reply by ErenCaldiran** [reply]
Will to be released "ONNX Runtime 1.15" support Jetpack 5.1.1 and so Jetson Orin Nano Developer Kit ?
